### ISSUE TITLE
Badges on implementation page

### ISSUE THREAD

**Issue by harrel56** [bowtie-json-schema/bowtie#926]
Bowtie badges are still kind of a hidden feature. Maybe if we displayed all the available badges on each implementation page it'd be more clear what is actually accessible. Anyone could just copy the url/link to such badge and use it on his own.

So each implementation would get:
- [![Supported spec](https://img.shields.io/endpoint?url=https%3A%2F%2Fbowtie-json-schema.github.io%2Fbowtie%2Fbadges%2Fjava-dev.harrel.json-schema%2Fsupported_versions.json)](https://img.shields.io/endpoint?url=https%3A%2F%2Fbowtie-json-schema.github.io%2Fbowtie%2Fbadges%2Fjava-dev.harrel.json-schema%2Fsupported_versions.json)
- [![Compliance](https://img.shields.io/endpoint?url=https%3A%2F%2Fbowtie-json-schema.github.io%2Fbowtie%2Fbadges%2Fjava-dev.harrel.json-schema%2Fcompliance%2FDraft_2020-12.json)](https://img.shields.io/endpoint?url=https%3A%2F%2Fbowtie-json-schema.github.io%2Fbowtie%2Fbadges%2Fjava-dev.harrel.json-schema%2Fcompliance%2FDraft_2020-12.json) - and one of those for each dialect/draft version the implementation supports.

@Julian ofc let me know what you think

**Comment by Julian** [comment]
This is kind of a duplicate of #211 but better written and a better suggestion :)

So yes I agree we shouldn't wait any longer and that putting this on the implementation page is a great idea!

**Comment by Julian** [comment]
After #927 is finished, sure!

**Comment by sd1p** [comment]
Hi, I am new to this project. I need some guidance to work on this issue. After reading the issue description carefully, I tried to implement a prototype for the required changes in the UI. Please provide a feedback if I am understanding issue correctly. 

 
![image](https://github.com/bowtie-json-schema/bowtie/assets/43758514/72e78ac7-f665-448c-926b-d84d9eadd7c5)
![image](https://github.com/bowtie-json-schema/bowtie/assets/43758514/62dff64c-cc7b-4f41-ae25-a5fb06ba8dd8)

**Comment by harrel56** [comment]
@sd1p The placement of the badges looks good but it's still missing the one with all supported dialects. 

But for the badges themself you should just use an image under link e.g. https://img.shields.io/endpoint?url=https%3A%2F%2Fbowtie-json-schema.github.io%2Fbowtie%2Fbadges%2Fjava-dev.harrel.json-schema%2Fsupported_versions.json. You can click the badges in the description to see the images. As you can see the link contains URI in query parameter - in that case it contains `java-dev.harrel.json-schema` implementation, if you replace it with different implementation ID you will get badge for that implementation.

Let me know if you have any other questions - and it'll be easier if you opened a PR (or at least provide some link to your branch with changes), that way I could see if it's going in a good direction

**Comment by sd1p** [comment]
Added a PR for this issue.
https://github.com/bowtie-json-schema/bowtie/pull/929
